tests/Drupal/Commenting/FunctionCommentUnitTest.inc.fixed

```diff
@@ -552,3 +552,21 @@ function test38($a, $b) {
 function test39(mixed $arg) {
   return $arg;
 }
+
+/**
+ * Test multiline comments with excess spaces in the @param annotation.
```

Comment:
the @ sign should also be removed here

Reply:
Fixed. That passed on my local as it was - I presume I mis-called phpunit somehow on that - any known easy way to mess that up?

Reply:
not sure how that could go wrong - now the coding standards are failing, I feel a bit bad for you now :-|

But after that I think it should really work.

Reply:
@NickDickinsonWilde I expect you know, but the coding standards used in the Coder module are a bit different from the rest of Drupal. What command were you using to the check the standards? If running locally then your phpunit will probably not check any standards.

Reply:
ah bother. I ran phpcs from a big project on that specific path. That's embarrassing. 
Thanks!